CHANGELOG.md

```diff
@@ -11,11 +11,13 @@
    - `LAYMAN_AUTHZ_MODULE=layman.authz.read_everyone_write_owner` (variable to remove)
       - `GRANT_CREATE_PUBLIC_WORKSPACE=` (new variable)
       - `GRANT_PUBLISH_IN_PUBLIC_WORKSPACE=` (new variable)
+- [#101](https://github.com/jirik/layman/issues/101) Change [LAYMAN_CLIENT_VERSION](doc/env-settings.md#LAYMAN_CLIENT_VERSION) from `v1.3.0` to `v1.4.0`
```

Comment:
Tuhle změnu nevidím v .env filech, ani nevidím daný release. Jeslti je to myšleno jen jako příprava, raději bych s ní počkal, až to bude implementováno.